src/index.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<body>
```

Comment:
Missing `<head>` tag.
You may add base markup to your document by typing `!` in an empty file in your editor and pressing `tab` button